doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -245,6 +245,16 @@ have to be observed for notations starting with a symbol, e.g., rules
 starting with “\ ``{``\ ” or “\ ``(``\ ” should be put at level 0. The list
 of Coq predefined notations can be found in the chapter on :ref:`thecoqlibrary`.
 
+.. warn:: Closed notations (i.e. starting and ending with a terminal symbol) should usually be at level 0 (default).
+   :name: closed-notation-not-level-0
+
+   It is usually better to put closed notations, that is the ones starting and ending with a terminal symbol, at level 0.
```

Comment:
Maybe can we also argue that it is a convention.

Reply:
We could indeed but I guess it's fine as it is (I mean it's not *just* an arbitrary convention).
For me this is ready to merge.